README.md

```diff
@@ -30,6 +30,10 @@ This app is still in the early stages of development and only some functionality
 ---
 
 # Local Development
+
+##  Node
```

Comment:
```suggestion
## Node
```

Reply:
Committed the Suggestion. They never teach things like this in school. Thats what we call real life experience. Thank you for the feedback